weblate/static/style-bootstrap.css

```diff
@@ -508,6 +509,7 @@ form + .login-label {
   border-radius: 0 5px 5px 5px;
   background-color: #e5f0f2;
   padding: 8px 10px;
+  word-wrap: break-word;
```

Comment:
Is this change really needed? In case it is, please submit it separately as this has nothing to do with comment editing, but affects comment rendering.

```suggestion
```

Reply:
Okay, looking at the commit I understand it, but it was a bit confusing to see it in editor related pull request :-).